isolate_example/lib/page_three.dart

```diff
@@ -11,6 +11,8 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+import 'dart:typed_data';
```

Comment:
There are `dart` imports in two places in this file, and they should be grouped together. Ctrl-Opt-O should sort them for you in Android Studio

Reply:
It looks like this is still an issue in the most recent revision. The `dart:` imports should be grouped together at the top.